samples/dashboards/demos/minimal/demo.js

```diff
@@ -1,115 +1,141 @@
-const CSVConnector = Dashboards.DataConnector.types.CSV;
+const csvData = document.getElementById('csv').innerText;
 
-const csvData = document.getElementById('csv').innerText,
-    connector = new CSVConnector({
-        csv: csvData,
-        firstRowAsNames: true
+async function setupDashboard() {
+    const board = Dashboards.board('container', {
+        dataPool: {
+            connectors: [{
+                name: 'Vitamin',
+                type: 'CSV',
+                options: {
+                    csv: csvData,
+                    firstRowAsNames: true
+                }
+            }]
+        },
+        editMode: {
+            enabled: true,
+            contextMenu: {
+                enabled: true,
+                items: ['editMode']
+            }
+        },
+        gui: {
+            layouts: [
+                {
+                    id: 'layout-1',
+                    rowClassName: 'custom-row',
+                    cellClassName: 'custom-cell',
+                    rows: [
+                        {
+                            cells: [
+                                {
+                                    id: 'dashboard-col-0',
+                                    width: '50%'
+                                },
+                                {
+                                    id: 'dashboard-col-1'
+                                },
+                                {
+                                    id: 'dashboard-col-12'
+                                }
+                            ]
+                        },
+                        {
+                            id: 'dashboard-row-1',
+                            cells: [
+                                {
+                                    id: 'dashboard-col-2',
+                                    width: '1'
+                                }
+                            ]
+                        }
+                    ]
+                }
+            ]
+        }
     });
 
-connector.load();
+    const connector = await board.dataPool.getConnector('Vitamin');
 
-Dashboards.board('container', {
-    connector,
-    editMode: {
-        enabled: true,
-        contextMenu: {
-            enabled: true,
-            items: ['editMode']
+    board.setComponents([{
```

Comment:
Is that the only way for now to use the data connectors with the board?

Reply:
For now, yes 😓